### DISCUSSION TITLE
Highlighting open folders

### DISCUSSION THREAD

**Discussion by mhgffqwoer** [nvim-neo-tree/neo-tree.nvim#1416] [Q&A]
I’m not familiar with Lua at all, and so I have a question: how to make an open area tint like in the picture.
<img width="152" alt="image" src="https://github.com/nvim-neo-tree/neo-tree.nvim/assets/119455427/d0d62526-bd36-4e8b-a2a4-d0b6a83995bd">

**Comment by pysan3** [comment]
The color does not come from whether the folder is opened or not,
but the color changes when there's a git modification in one of the children files.

I assume some changes were make to a file under `src/components/Header/...`.
